src/status_im2/contexts/wallet/common/temp.cljs

```diff
@@ -73,3 +64,17 @@
    :account-name        "Account 1"
    :account             :default
    :customization-color :blue})
+
+(defn keypair-string
+  [full-name]
+  (let [first-name (first (string/split full-name #" "))]
+    (i18n/label :t/keypair-title {:name first-name})))
```

Comment:
Not for this PR:
Probably we need to abstract the `(first (string/split full-name #" "))` in wallet utils. WDYT?